.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,4 +1,33 @@
-
+<!--
+Thanks for sending a pull request!  Here are some tips for you:
+  1. If this is your first time, please read our [contributor guidelines](https://github.com/elyra-ai/community/blob/master/contributing.md)
+    1.1 Please follow the [7 rules for a great commit message](https://github.com/elyra-ai/community/blob/master/contributing.md#creating-a-pull-request)
+  2. If the PR is unfinished, leave it as 'DRAFT', add '[WIP]' in your PR title, e.g., '[WIP] Your PR title ...' or use the 'status:Work in Progress' label.
```

Comment:
I think just using the WIP label should suffice. Dont know about others, but I dont really like the `[WIP]' in the title

Reply:
I agree. I prefer to use draft mode to protect against premature merges as well. The wip label is helpful for project management though.

Reply:
WIP and labels are visible outside of the pr, labels allow for easy filtering, the draft is the hardest to use, but as long as there is some way of knowing, I am ok.